apis/kubedb/v1alpha1/proxysql_types.go

```diff
@@ -0,0 +1,117 @@
+package v1alpha1
+
+import (
+	"github.com/appscode/go/encoding/json/types"
+	apps "k8s.io/api/apps/v1"
+	core "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	mona "kmodules.xyz/monitoring-agent-api/api/v1"
+	ofst "kmodules.xyz/offshoot-api/api/v1"
+)
+
+const (
+	ResourceCodeProxySQL     = "psql"
```

Comment:
Is this the right short code? `psql` is more for postgresql

Reply:
resolved